addons/binding/pom.xml

```diff
@@ -17,6 +17,7 @@
   <name>openHAB Bindings</name>
 
   <modules>
+    <!--
```

Comment:
please remove the comments and just add your binding module.

Reply:
this had already been fixed. Maybe a mistake during rebase...
Will fix this again.